src/main/java/org/jabref/gui/actions/JabRefAction.java

```diff
@@ -77,10 +76,9 @@ private void trackExecute(String actionName) {
     }
 
     private void trackUserActionSource(String actionName, Sources source) {
-        Map<String, String> properties = new HashMap<>();
-        Map<String, Double> measurements = new HashMap<>();
-        properties.put("Source", source.toString());
-
-        Globals.getTelemetryClient().ifPresent(telemetryClient -> telemetryClient.trackEvent(actionName, properties, measurements));
+        Globals.getTelemetryClient().ifPresent(telemetryClient -> telemetryClient.trackEvent(
+                actionName,
+                Map.of("Source", source.toString()),
+                Map.of()));
```

Comment:
The empty map (properties) should come before the non-empty map (measurements).

Reply:
I just checked the JavaDoc (again)

![grafik](https://user-images.githubusercontent.com/1366654/124518586-1a199480-dde7-11eb-9243-6e05548d29db.png)

I thought, I would like to search and filter for the "Source". 

If I swap the parameters, I get the following error:

![grafik](https://user-images.githubusercontent.com/1366654/124518610-27368380-dde7-11eb-937f-d166e057fbef.png)

Reply:
Yes, you are right. Not sure why I thought the order was wrong